osu.Game.Rulesets.Osu/Skinning/Argon/ArgonMainCirclePiece.cs

```diff
@@ -176,15 +184,27 @@ private void updateStateTransforms(DrawableHitObject drawableHitObject, ArmedSta
                         using (BeginDelayedSequence(flash_in_duration / 12))
                         {
                             outerGradient.ResizeTo(OUTER_GRADIENT_SIZE * shrink_size, resize_duration, Easing.OutElasticHalf);
-                            outerGradient
-                                .FadeColour(Color4.White, 80)
-                                .Then()
-                                .FadeOut(flash_in_duration);
+
+                            if (showFlash)
+                            {
+                                outerGradient
+                                    .FadeColour(Color4.White, 80)
+                                    .Then()
+                                    .FadeOut(flash_in_duration);
+                            }
+                            else
+                            {
+                                outerGradient
+                                    .FadeColour(Color4.White, flash_in_duration * 8)
+                                    .FadeOut(flash_in_duration * 2);
+                            }
                         }
 
-                        flash.FadeTo(1, flash_in_duration, Easing.OutQuint);
+                        if (showFlash)
+                            flash.FadeTo(1, flash_in_duration, Easing.OutQuint);
+
+                        this.FadeOut(showFlash ? fade_out_time : fade_out_time / 2, Easing.OutQuad);
```

Comment:
The thing that makes me doubt is the persistence of the *"after circle"*. This is the line that needs to be tweaked. After playing with it for a while, I've found a sweet spot with `fade_out_time / 2.75` that makes the animation the least persistent but keeps the "bounciness" of the animation.

Reply:
I'm not willing to go that far sorry. That is not related to hit lighting.